src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -26,6 +26,7 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#nullable enable
```

Comment:
When doing a piece of a partial file, please do all pieces, e.g. please include CultureInfo.Windows.cs in this PR.

Reply:
it's hard to iterate on that locally since we bulk do it (we'd have to locally build it for all platforms) - I'd prefer if we had separate passes on them

Reply:
> we'd have to locally build it for all platforms

That's trivial:
build windowsmscorlib debug
build linuxmscorlib debug
build osxmscorlib debug

> I'd prefer if we had separate passes on them

I'd prefer it not.  Having a type in a half-converted state is difficult to reason about, especially when some public surface area exists in each partial and should be the same across them.

Reply:
this didn't work for me when I built on linux

Reply:
> this didn't work for me when I built on Linux

I've not tried these on Linux.  They all work on Windows.

Regardless of whether they work or not, we should not merge changes to partial files without scrubbing all portions of the partial file.

Reply:
fair enough